nulog-inference-service/NulogServer.py

```diff
@@ -84,7 +84,7 @@ def predict(self, logs: List[str]):
         output = []
         for log in logs:
             tokens = self.parser.tokenize_data([log], isTrain=False)
-            if len(tokens[0]) < MIN_LOG_TOKENS:
+            if len(tokens[0]) < min_log_tokens:
```

Comment:
can we move min_log_tokens to the constructor of this class, so it can get this value by `self.min_log_tokens`?
or perhaps `MIN_LOG_TOKENS` can be imported at the beginning of this script from the main script.